03-MVC-AngularJS/module.js

```diff
@@ -0,0 +1,113 @@
+var myApp = angular.module('myApp', []);
+
+myApp.factory('Storage', function () {
+	return {
+		get: function(string) {
+			var json_obj = localStorage.getItem(string);
+			return JSON.parse(json_obj);
+		},
+		post: function(list, string) {
+			var json_obj = JSON.stringify(list);
+			localStorage.setItem(string, json_obj);
+		}
+	};
+});
+ 
+myApp.service('Storage_Serv', function(Storage){
```

Comment:
this service is unnecessary I think, you can directly use the factory to interact with storage